.adr/0010-interest-bearing-channel.md

```diff
@@ -0,0 +1,128 @@
+4. Flexibility: 🙂 It's relatively easy in this model to adjust fees on the fly and also to adjust based on the number of hops involved in the virtual fund.
+5. Ergonomics:
+   - AppDev: 😀 no effect on channel application development
+   - Wallet: 😐 adds some complexity to the virtual-fund / virtual-defund operations
+   - Users: 🙁 Networking fees originating at the service consumer-side can potentially introduce extra UI hurdles (eg, think of setting gas fees for an L1 transaction). This can be obfuscated from the user by building network fees into the total service fee, but this creates potential for information asymmetries in the marketplace which aren't to the user's benefit.
+6. software-lock: 🙁 Modifications to the adjudicator are the hardest lock-in possible for the network. If this implementation is revealed to be broken, or unfit for purpose, then migration to a separate in-protocol implementation would require a redeployment of the adjudicator, hard shut-down of the entire running SCN, etc. Software-lock could be improved by, eg, implementing claim via upgradable contract, but this introduces tradeoffs in the credible trustlessness of the network.
+
+Meta commentary: sketches toward implementation of this model showed that the engineering, security and auditability effort would be reasonably high for this option.
+
+## Interest Bearing Application Channels
+
+> According to [this document](https://www.notion.so/statechannels/Space-Estimates-a7b26bb1b7204f2b85d536bf5930994c) and other related estimates, we expect _capital lock-up_ to be the dominant constraint for intermediaries in a nitro state channel network - far exceeding the costs of hardware & networking. According to analysis in this document, we expect competition for capital from various interest-yielding options.
+>
+> Given that `value*time` is be the dominating cost, the proposed model here is for liquidity providers and service providers to run interest-bearing ledger channels. The interest calculation is enforced by an application channel. Service providers gain access to the network (of service consumers) in exchange for an agreed interest rate paid on the liquidity provider's deposits. Some particular properties:
+>
+> - service providers who fail to do any business will not (can not possibly) pay any fees, since they do not deposit into their ledger channels. Interest collected by intermediaries is "clawed back" from service provider earnings.
+> - creation of virtual channels in this model is _free_, both in terms of a flat networking fee and in terms of capital lockup inside of intermediary channels. Justification for this is that
+>   - we expect the networking costs to be effectively negligible vs the capital costs
+>   - intermediary channels rae by definition operated by our liquidity providers, who are the best positioned participants to price these operations into their offerings to service providers
```

Comment:
```suggestion
>   - intermediary channels are by definition operated by our liquidity providers, who are the best positioned participants to price these operations into their offerings to service providers
```

Querying your use of "our" here... I assume a literary "our" rather than implying the LPs belong to statechannels.org or something?

Reply:
Oh, yes. The "royal" ours. In fact, the sentence reads at least as well, and less ambiguously, with the word removed. I'll do that.